src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/InterventionDTO.kt

```diff
@@ -5,13 +5,30 @@ import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Interve
 data class InterventionDTO(
   val title: String,
   val description: String,
+  val pccRegions: List<String>,
+  val serviceCategory: String,
+  val serviceProvider: String,
+  val minimumAge: Int,
+  val maximumAge: Int,
+  val allowsFemale: Boolean,
+  val allowsMale: Boolean,
 ) {
   companion object {
 
-    fun from(intervention: Intervention): InterventionDTO {
+    fun from(intervention: Intervention, pccRegions: List<String>): InterventionDTO {
+      val contract = intervention.dynamicFrameworkContract
+      val contractEligibility = contract.contractEligibility
+
       return InterventionDTO(
         title = intervention.title,
         description = intervention.description,
+        pccRegions = pccRegions,
+        serviceCategory = contract.serviceCategory.name,
+        serviceProvider = contract.serviceProvider.name,
+        minimumAge = contractEligibility.minimumAge,
+        maximumAge = contractEligibility.maximumAge,
+        allowsFemale = contractEligibility.allowsFemale,
+        allowsMale = contractEligibility.allowsMale,
```

Comment:
The code hints that these belong together -- it's probably ok to model them as
```json
"eligibility": {
  "minimumAge": 18,
  "maximumAge": null,
  "allowsMale": true,
  "allowsFemale": true,
}
```
What do you think @lawrence-forooghian, @Gweaton? What'd be simpler to understand and use from a UI perspective?

Reply:
That's a good point, hadn't thought of that, I think your suggestion looks good @sldblog